api/handlers/dbinit.js

```diff
@@ -0,0 +1,97 @@
+const Pool = require('pg').Pool;
+
+
+const dbInitLambdaFunction = async (event) => {
+    const response = {
+        statusCode: 200,
+        body: JSON.stringify('Lambda function executed db commands successfully')
+    }
+    try{
+        const pool = new Pool({
+            user: 'masteruser',
+            password: "ecommdbpw",
+            host: 'ecommdb.covnxnivrrc5.ap-southeast-2.rds.amazonaws.com',
+            database: "ecommdb",
+            port: 5432,
+            ssl: true
+        })
+        await pool.query(`GRANT ALL PRIVILEGES ON DATABASE ecommdb TO masteruser`)
+        await pool.query('DROP TABLE IF EXISTS carts');
+        await pool.query('DROP TABLE IF EXISTS products');
+        await pool.query('DROP TABLE IF EXISTS users');
+        await pool.query(`CREATE TABLE products (
+            id SERIAL PRIMARY KEY,
+            product_name VARCHAR NOT NULL,
+            product_description TEXT NOT NULL,
+            category VARCHAR NOT NULL,
```

Comment:
There should be an index on `category` because there are SQL queries with WHERE clauses on `category`.